admin_manual/enterprise/external_storage/windows-network-drive_configuration.rst

```diff
@@ -264,6 +270,31 @@ See `Configuring wnd:listen to run as a service
 in the documentation wiki for tips on running the listener as a service via
 cron, and by creating a `systemd`_ startup script.
 
+
+.. _password-options-label:
+
+Password Options
+----------------
+
+There are three ways to supply a password:
+
+#. Interactively in response to a password prompt.
+#. Sent as a parameter to the command, e.g., ``occ wnd:listen host share username password``.
+#. Read from a file, using the ``--password-file`` switch to specify the file to read from. 
+
+.. note::
+   If you use the ``--password-file`` switch, the entire contents of the file will be used for the password, so please be careful with newlines.
+   You can also use ``--password-file=-`` to read the password from the standard input. However, you might need to press ``Ctrl+D`` to close the input and send it to the command.
+
+.. warning::
+   If using ``--password-file`` make sure that the file is inaccessible from the web in order to prevent tampering or leaking of the information. The password won't be leaked to any other user using ``ps``.
```

Comment:
the file needs to be readable by the apache / www-data user, which is the one that will execute the command.